plugins/tiddlywiki/filesystem/filesystemadaptor.js

```diff
@@ -122,14 +122,22 @@ FileSystemAdaptor.prototype.loadTiddler = function(title,callback) {
 };
 
 /*
-Delete a tiddler and invoke the callback with (err)
+	Delete a tiddler and invoke the callback with (err)
+	@params title title of tiddler to delete
```

Comment:
We don't use jsdoc style comments

Reply:
Can there be a post in Github Discussion talking about this? Currently $tw API doc is incomplete, which makes there being some obstacles in plugin development.